app/models/creator.rb

```diff
@@ -0,0 +1,115 @@
+class Creator < ActiveRecord::Base
+  has_many :content
+  has_and_belongs_to_many :users
+  validates :p_id, :provider, :username, presence: true
+
+  include HTTParty
+  LIMIT_PER_PAGE = 10
+
+  #method to configure twitter
+  def twit
+    Seemore::Application.config.twitter
+  end
```

Comment:
You have this a few places. You could create a helper method or a Twitter class (that you'd put in /lib) and use that, rather than having several `twit` methods that you'd need to maintain.